docs/decisions/0011-producing-to-event-bus-via-settings.rst

```diff
@@ -0,0 +1,106 @@
+11. Enable producing to event bus via settings
+##############################################
+
+Status
+******
+
+**Provisional** 2023-05-23
+
+Context
+*******
+
+Since the initial implementation of the event bus allowed only a single event
```

Comment:
OEP-19 requests that hard line breaks be avoided in paragraphs in rST files: https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0019-bp-developer-documentation.html#best-practices